lntest/itest/rpc.go

```diff
@@ -0,0 +1,168 @@
+// +build rpctest
```

Comment:
Any reason for the `rpc.go` filename? Was expecting `macaroons.go`.

Reply:
I wasn't sure how many more macaroon tests we would add so I thought something more generic would maybe be better so other tests could be moved to that file. But `rpc` also doesn't really make sense since there aren't any other tests for just the gRPC level interface. Changed it to `macaroons`.